mailet/standard/src/main/java/org/apache/james/transport/mailets/ClamAVScan.java

```diff
@@ -266,8 +266,8 @@ public String getMailetInfo() {
      * Initializer for property debug.
      */
     protected void initDebug() {
-        String debugParam = getInitParameter("debug");
-        setDebug((debugParam == null) ? false : Boolean.valueOf(debugParam));
+        Boolean debugParam = getInitParameter("debug", Boolean.class);
+        setDebug((debugParam == null) ? false : debugParam);
```

Comment:
This can be `Optional.ofNullable(debugParam).orElse(false)` which would be more functional friendly and more readable.